File: VECTOAux/VectoAuxiliaries/Hvac/DeleteColumn.vb
Imports System.Windows.Forms


Public Class DeleteColumn
  Inherits DataGridViewButtonColumn



   Public  Sub new()

     MyBase.New()
     Me.CellTemplate = New DeleteCell()

   End Sub




End Class

Public Class DeleteAlternatorColumn
  Inherits DataGridViewButtonColumn



   Public  Sub new()

     MyBase.New()
     Me.CellTemplate = New DeleteAlternatorCell()

   End Sub




End Class

File: VECTOAux/BusAuxiliaries/DownstreamModules/Impl/HVAC/DeleteColumn.cs
using System.Windows.Forms;

namespace TUGraz.VectoCore.BusAuxiliaries.DownstreamModules.Impl.HVAC {
	public class DeleteColumn : DataGridViewButtonColumn
	{
		public DeleteColumn() : base()
		{
			this.CellTemplate = new DeleteCell();
		}
	}

	public class DeleteAlternatorColumn : DataGridViewButtonColumn
	{
		public DeleteAlternatorColumn() : base()
		{
			this.CellTemplate = new DeleteAlternatorCell();
		}
	}
}